Version bump for 1.10.0 release + update readme (#997)
Flow-Launcher/Flow.Launcher: README.md

- Switch easily between Everything and Windows Search to take advantage of both search engines (remember to remove existing Everything plugin).
- Use features available to both Everything and Explorer plugins

### Date & Time Display In Search Window

<img src="https://user-images.githubusercontent.com/6903107/207159348-8b0c7a2b-0836-4764-916b-e0236087f7f3.png" width="400">

- Display the date and time when the search window is triggered.

### Drag & Drop
<img src="https://user-images.githubusercontent.com/6903107/207159486-1993510f-09f2-4e33-bba7-4ca59ca1bc5a.png" width="500">

- Drag an item to Discord or computer location.
- The target program determines whether the drop is to copy or move the item (can change via CTRL or Alt), and the operation is displayed on the mouse cursor.

### Custom Shortcut

<img src="https://user-images.githubusercontent.com/6903107/207170354-db1a98a6-3959-4341-82ed-0c417bdb121c.png" width="300">
<img src="https://user-images.githubusercontent.com/6903107/207170525-3e99ad93-e8b7-4b39-b9b3-8fbeb443ed26.png" width="300">

- New shortcut functionality to set additional action keywords or search terms.

> ### Impvroved Program Plugin
> - Added Index PATH
> - Support .url files. so flow can search steam/epic games in now.
> - Improved Indexing UWP.

Author: ```suggestion
### Impvroved Program Plugin
- PATH is now indexed
- Support for .url files, flow can now search installed steam/epic games.
- Improved UWP indexing.
```

> ### Improved Memory usage
> - Fixed a memory leak and reduced overall memory usage.
> ### Improved Plugin / Plugin Store
> - Add Searchbox. you can search plugins in plugin / plugin store.
> - Add Category in plugin store.(Installed, New...)
> 
> ### Improve non-c# plugin panel design
> <img src="https://user-images.githubusercontent.com/6903107/207166078-871a17aa-5ab4-4808-8d8f-12b7cb66ea89.png" width="450">
> 
> 
> - The design has been adjusted to suit the overall design feel.
> - Changed the complex bottom information simply.
> 
> 🚂<a href="https://github.com/Flow-Launcher/Flow.Launcher/releases">Full Changelogs</a>

Author: ```suggestion
### Improved Memory Usage
- Fixed a memory leak and reduced overall memory usage.

### Improved Plugin / Plugin Store
- Search plugins in the Plugin Store and existing plugin tab.
- Categorised sections in Plugin Store to easily see new and updated plugins.

### Improved Non-C# Plugin's Panel Design
<img src="https://user-images.githubusercontent.com/6903107/207166078-871a17aa-5ab4-4808-8d8f-12b7cb66ea89.png" width="450">

- The design has been adjusted to align to the overall look and feel of flow.
- Simplified the information displayed on buttons

🚂<a href="https://github.com/Flow-Launcher/Flow.Launcher/releases">Full Changelogs</a>
```

<p align="center"> <sub>Remember to star it, flow will love you more :)</sub></p>

<img src="https://user-images.githubusercontent.com/6903107/144858082-8b654daf-60fb-4ee6-89b2-6183b73510d1.png" width="100%">

## 🎅 New Features🤶
### Preview Panel

> <img src="https://user-images.githubusercontent.com/6903107/207159213-662999d3-2c18-4256-b473-c417efca0069.png" width="400">
> 
> - Use the F1 key to open/hide the preview panel.
> - Media files will be displayed as large images, otherwise a large icon and entire path will be displayed.
> - Turn on preview permanently via Settings (Always Preview).
> - Use hotkeys (Ctrl+Plus,Minus / Ctrl+],[) to adjust flow's search window width and height quickly if the preview area is too narrow.
> - This feature is currently in its early stages.
> 
> ### Everything plugin merged to Explorer
> <img src="https://user-images.githubusercontent.com/6903107/207171178-78912e76-31c2-4ba2-b3cf-de4b0b5e5691.png" width="300">
> 
> - Everything's features are now included in the Explorer plug-in.
> - If you are already using everything plugin, you can only enable the explorer.
> - If you are using windows search indexer, add-ons are available from the Everything plug-in.
> 
> ### Clock & Dates in Search box
> 
> <img src="https://user-images.githubusercontent.com/6903107/207159348-8b0c7a2b-0836-4764-916b-e0236087f7f3.png" width="400">
> 
> - Display Time & dates in searchbox.
> 
> ### Drag and Drop
> <img src="https://user-images.githubusercontent.com/6903107/207159486-1993510f-09f2-4e33-bba7-4ca59ca1bc5a.png" width="500">
> 
> - You can drag an item to a discord or desktop.
> - The drop target program determines whether to copy or move (You can change it CTRL or Alt), and is displayed in the mouse cursor.
> 
> ### Custom Shortcut
> 
> <img src="https://user-images.githubusercontent.com/6903107/207170354-db1a98a6-3959-4341-82ed-0c417bdb121c.png" width="300">
> <img src="https://user-images.githubusercontent.com/6903107/207170525-3e99ad93-e8b7-4b39-b9b3-8fbeb443ed26.png" width="300">
> 
> - Added shortcut functionality. This allows you to set additional action keywords.

Author: ```suggestion
<img src="https://user-images.githubusercontent.com/6903107/207159213-662999d3-2c18-4256-b473-c417efca0069.png" width="400">

- Use the F1 key to open/hide the preview panel.
- Media files will be displayed as large images, otherwise a large icon and entire path will be displayed.
- Turn on preview permanently via Settings (Always Preview).
- Use hotkeys (Ctrl+Plus,Minus / Ctrl+],[) to adjust flow's search window width and height quickly if the preview area is too narrow.
- This feature is currently in its early stages.

### Everything Plugin Merged Into Explorer
<img src="https://user-images.githubusercontent.com/6903107/207171178-78912e76-31c2-4ba2-b3cf-de4b0b5e5691.png" width="300">

- Switch easily between Everything and Windows Search to take advantage of both search engines (remember to remove existing Everything plugin).
- Use features available to both Everything and Explorer plugins

### Date & Time Display In Search Window

<img src="https://user-images.githubusercontent.com/6903107/207159348-8b0c7a2b-0836-4764-916b-e0236087f7f3.png" width="400">

- Display the date and time when the search window is triggered.

### Drag & Drop
<img src="https://user-images.githubusercontent.com/6903107/207159486-1993510f-09f2-4e33-bba7-4ca59ca1bc5a.png" width="500">

- Drag an item to Discord or computer location.
- The target program determines whether the drop is to copy or move the item (can change via CTRL or Alt), and the operation is displayed on the mouse cursor.

### Custom Shortcut

<img src="https://user-images.githubusercontent.com/6903107/207170354-db1a98a6-3959-4341-82ed-0c417bdb121c.png" width="300">
<img src="https://user-images.githubusercontent.com/6903107/207170525-3e99ad93-e8b7-4b39-b9b3-8fbeb443ed26.png" width="300">

- New shortcut functionality to set additional action keywords or search terms.
```

<p align="center"> <sub>Remember to star it, flow will love you more :)</sub></p>

<img src="https://user-images.githubusercontent.com/6903107/144858082-8b654daf-60fb-4ee6-89b2-6183b73510d1.png" width="100%">

## 🎅 New Features🤶
### Preview Panel
<img src="https://user-images.githubusercontent.com/6903107/207159213-662999d3-2c18-4256-b473-c417efca0069.png" width="400">

- You can use the F1 key to open/hide the preview panel.
- Media files display large images.
- If it is not a media file, a large icon and the entire path are displayed.

> - You can always turn it on (Always Preview) in Settings.

Author: ```suggestion
- You can turn it on permanently via Settings (Always Preview).
```

> - You can use the F1 key to open/hide the preview panel.
> - Media files display large images.
> - If it is not a media file, a large icon and the entire path are displayed.
> - You can turn it on permanently via Settings (Always Preview).
> - Added hotkeys (Ctrl+Plus,Minus / Ctrl+],[) to quickly change flow window's width and height. If you feel that the preview area is narrow, adjust it quickly.
> - This feature is currently in its early stages.

Author: ```suggestion
- Use the F1 key to open/hide the preview panel.
- Media files will be displayed as large images, otherwise a large icon and entire path will be displayed.
- Turn on preview permanently via Settings (Always Preview).
- Use hotkeys (Ctrl+Plus,Minus / Ctrl+],[) to adjust flow's search window width and height quickly if the preview area is too narrow.
- This feature is currently in its early stages.
```

| <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>Enter</kbd>            | Run as admin                                 |
| <kbd>↑</kbd><kbd>↓</kbd>                                     | Scroll up & down                             |
| <kbd>←</kbd><kbd>→</kbd>                                     | Back to result / Open Context Menu           |
| <kbd>Ctrl</kbd> +<kbd>O</kbd> , <kbd>Shift</kbd> +<kbd>Enter</kbd> | Open Context Menu                      |
| <kbd>Tab</kbd>                                               | Autocomplete                                 |
| <kbd>F1</kbd>                                                | Toggle Preveiw Panel                         |
| <kbd>Esc</kbd>                                               | Back to results / hide search window         |
| <kbd>Ctrl</kbd> +<kbd>C</kbd>                                | Copy the actual folder / file                |
| <kbd>Ctrl</kbd> +<kbd>I</kbd>                                | Open Settings window                         |
| <kbd>F5</kbd>                                                | Reload all plugin data                       |

> | <kbd>Ctrl</kbd> + <kbd>F12</kbd>                             | Toggle Game Mode                             |
> | <kbd>Ctrl</kbd> + <kbd>+</kbd>,<kbd>-</kbd>                  | Quick Change Maximum Result Shown            |
> | <kbd>Ctrl</kbd> + <kbd>[</kbd>,<kbd>]</kbd>                  | Quick Change Window Width                    |

Author: ```suggestion
| <kbd>Ctrl</kbd> + <kbd>F12</kbd>                             | Toggle Game Mode when in search window.          |
| <kbd>Ctrl</kbd> + <kbd>+</kbd>,<kbd>-</kbd>                  | Quickly change maximum results shown            |
| <kbd>Ctrl</kbd> + <kbd>[</kbd>,<kbd>]</kbd>                  | Quickly change search window width                    |
```

  <a href="https://appwrite.io">
    <img src='https://appwrite.io/images-ee/press/logo-1.svg' width="25%"/>
  </a>
  <br />

</p>

<p align="center">
  <a href="https://github.com/itsonlyfrans">
    <img src='https://avatars.githubusercontent.com/u/46535667?v=4' width="10%"/>
          <br /><sub><b>itsonlyfrans</b></sub>
  </a>
</p>

> ## Media's Recommends
> - <a href="https://dev.to/appwrite/appwrite-loves-open-source-why-i-chose-to-support-flow-launcher-54pj"> Why I Chose to Support Flow-Launcher</a> - appwrite
> - <a href="https://www.softpedia.com/get/System/Launchers-Shutdown-Tools/Flow-Launcher.shtml">SOFTPEDIA EDITOR'S PICK</a>

Author: ```suggestion
## Mentions
- <a href="https://dev.to/appwrite/appwrite-loves-open-source-why-i-chose-to-support-flow-launcher-54pj"> Why I Chose to Support Flow-Launcher</a> - appwrite
- <a href="https://www.softpedia.com/get/System/Launchers-Shutdown-Tools/Flow-Launcher.shtml">SOFTPEDIA EDITOR'S PICK</a>
```
